Tests/Unit/DSL/Aggregation/RangeAggregationTest.php

```diff
@@ -118,24 +120,93 @@ public function getToArrayData()
             ],
         ];
 
-        $out[] = [
-            $aggregation,
-            $result,
+        $this->assertEquals($result, $aggregation->toArray());
+    }
+
+    /**
+     * Tests getType method.
+     */
+    public function testRangeAggregationGetType()
+    {
+        $agg = new RangeAggregation('foo');
+        $result = $agg->getType();
+        $this->assertEquals('range', $result);
+    }
+
+    /**
+     * Data provider for testRangeAggregationRemoveRangeByKey(), testRangeAggregationRemoveRange().
+     *
+     * @return array
+     */
+    public function testRangeAggregationDataProvider()
```

Comment:
Data providers should not only give expected results, but also data which you would give to objects/process and then check if the expected data matches with returned from object/process.